plutus-pab/src/Cardano/Wallet/Types.hs

```diff
@@ -62,14 +62,17 @@ data Config =
     deriving (Show, Eq, Generic)
     deriving anyclass (FromJSON, ToJSON)
 
-newtype WalletMsg = StartingWallet Port
+data WalletMsg = StartingWallet Port
+                  | ChainClientMsg Text
```

Comment:
odd formatting

Reply:
uh, yeah. I do wish `stylish-haskell` was  giving me less leeway when i do silly things like that